src/app/views/onboarding/step2.less

```diff
@@ -1,117 +1,125 @@
 #step2 {
   .apps-container {
+    display: -ms-flex;
+    display: -webkit-flex;
     display: flex;
```

Comment:
You don't need to add browser specific instructions, we add them at build time using [postcss](https://github.com/postcss/postcss) which is using [autoprefixer](https://github.com/postcss/autoprefixer).